src/TagsInput/components/InputField.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+
+import * as React from 'react';
+import { View, TextInput, Text } from 'react-native';
+import type { ViewLayoutEvent } from 'react-native/Libraries/Components/View/ViewPropTypes';
+import { StyleSheet } from '../../PlatformStyleSheet';
+
+type Props = {|
+  +onChangeText: (value: string) => void,
+  +value: string,
+  +onLayout: (e: ViewLayoutEvent) => void,
+  +maxWidth: ?number | ?string,
+  +minWidth: number,
+  +fontSize: number,
+  +disabled: ?boolean,
+  +placeholder: ?string,
+  +forwardRef: (el: any) => void,
+|};
+
+type State = {|
+  componentWidth: number | string,
```

Comment:
```suggestion
  componentWidth: number | 'auto',
```

That's the only string that's allowed, right?

Reply:
👍

Reply:
I cannot see it added now?

Reply:
fixed